packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -684,11 +687,18 @@ class _LineItemState extends State<LineItem> {
           if (response is! InstanceRef) return;
           final variable = Variable.fromRef(response);
           await _debuggerController.buildVariablesTree(variable);
+          if (_hasMouseExited) return;
+          _hoverCard?.remove();
           _hoverCard = HoverCard(
-            contents: Material(
-              child: ExpandableVariable(
-                debuggerController: _debuggerController,
-                variable: ValueNotifier(variable),
+            contents: SingleChildScrollView(
+              child: Container(
+                constraints: const BoxConstraints(maxHeight: 250.0),
```

Comment:
make 250.0 a const var on the theme. `maxHoverCardHeight` or similar

Reply:
Done. PTAL.